src/App.tsx

```diff
@@ -84,6 +122,33 @@ function App() {
       <div className="movie-details-container">
         <MovieDetails movie={movie} />
       </div>
+
+      <button onClick={openDialogAdd}>Add Movie</button>
+      {isOpenAdd && (
+        <Dialog title="ADD MOVIE" onClose={closeDialogAdd}>
+          <MovieForm onSubmit={onSubmitMovie} />
+        </Dialog>
+      )}
+
+      <button onClick={openDialogEdit}>Edit Movie</button>
+      {isOpenEdit && (
+        <Dialog title="EDIT MOVIE" onClose={closeDialogEdit}>
+          <MovieForm initialMovie={movie} onSubmit={onSubmitMovie} />
+        </Dialog>
+      )}
+
+      <button onClick={openDialogDeleteConfirmation}>Delete Movie</button>
+      {isOpenDeleteConfirmation && (
+        <Dialog title="DELETE MOVIE" onClose={closeDialogDeleteConfirmation}>
+          <p>Are you sure you want to delete this movie?</p>
```

Comment:
From UX perspective it is important to show the title of movie to delete.

Are you sure you want to delete <movie title>?